.versioningignore

```diff
@@ -1,6 +1,8 @@
 .env
 .venv
 venv
+env2.1
+env2.1/
```

Comment:
do we need it?

Reply:
My idea here was to create a config and an install script that will enable us to have both "env" with legacy torch 2.0 and "env2.1" with torch 2.1. If I don't mark it in all the ignore files, black will try to reformat all code from these libraries.